Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -147,16 +155,30 @@ class RubricCircleView: UIView {
 
         UIView.animate(withDuration: 0.2, delay: delay, usingSpringWithDamping: 0.25, initialSpringVelocity: 6.0, options: [.allowUserInteraction, .curveEaseInOut], animations: {
             self.currentlySelectedButton?.transform = CGAffineTransform.identity
-            buttonToAnimate.transform = self.selectedButtonTransform
+            buttonToAnimate?.transform = self.selectedButtonTransform
         }, completion: { _ in
             self.currentlySelectedButton = buttonToAnimate
             completionHandler()
         })
     }
 
+    func updateButtons(rubric: RubricViewModel) {
+        self.rubric = rubric
+        if rubric.isCustomAssessment {}
```

Comment:
should there be an `else` after this clause, or should we just remove it since it doesn't do anything?